# 给定下面代码片段

```
	try {
		method();
		System.out.println("Hello World");
	} catch (ArrayIndexOutOfBoundsException e) {
		System.out.println("Exception1");
	} catch (Exception e) {
		System.out.println("Exception2");
	} finally{
		System.out.println("Think You");			
	}
```
#　如果method抛出一个NullPointerException的异常，会显示( )

A Hello World   
B Exception1  
C Exception2  
D Think You

# 参考答案

C,D;

---